docs/src/getting_started/Android/android.md

```diff
@@ -16,9 +16,13 @@ We want to make setting up Android Studio to work with Crux really easy. As time
 The first thing we need to do is create a new Android app in Android Studio.
 
 Open Android Studio and create a new project, for "Phone and Tablet", of type
-"Empty Compose Activity (Material3)". In this walk-through, we'll call it
-"Counter", use a minimum SDK of API 34, and save it in a directory called
-`Android`.
+"Empty Activity". In this walk-through, we'll call it "Counter"
+
+* "Name": `Counter`
+* "Package name": `com.example.simple_counter`
```

Comment:
I think this is important, e.g. if I chose a different namespace like `com.random_tutorials.simple_counter` you'd then have a problem later as the `shared` library will be in the `com.example.` namespace? (Maybe I'm wrong, I'm not an Android/Java dev 😅)